packages/realm-web/src/transports/AuthenticatedTransport.test.ts

```diff
@@ -84,4 +84,30 @@ describe("AuthenticatedTransport", () => {
             },
         ]);
     });
+
+    it("returns an AuthenticatedTransport when prefixed", async () => {
+        const baseTransport = new MockTransport([{}]);
+        const transport = new AuthenticatedTransport(baseTransport, {
+            currentUser: { accessToken: "my-access-token" } as Realm.User,
+        });
+        const prefixedTransport = transport.prefix("/prefixed-path");
+        expect(prefixedTransport).to.be.instanceOf(AuthenticatedTransport);
+        // Send a request
+        await prefixedTransport.fetch({
+            method: "GET",
+            path: "/w00t",
+        });
+        // Expect something of the request
+        expect(baseTransport.requests).deep.equals([
+            {
+                method: "GET",
+                url: "http://localhost:1337/prefixed-path/w00t",
+                headers: {
+                    Authorization: "Bearer my-access-token",
+                    Accept: "application/json",
```

Comment:
You could spread `DEFAULT_HEADERS` here, so changes wont break this testing that they gets set. (and only checks that the `Authorization` header is added)

Reply:
I think something like https://github.com/realm/realm-js/blob/kh/web/fetch-location-metadata/packages/realm-web/src/App.test.ts#L212 covers this.